torchsynth/module.py

```diff
@@ -544,15 +548,6 @@ def noise_of_length(self, audio_in: Signal) -> Signal:
             return torch.rand_like(audio_in) * 2 - 1
 
 
-class Identity(SynthModule):
```

Comment:
So I don't think this was working the way we intended it to. In the case of a VCO in Voice, we would be passing through the pitch modulation envelope which would then be summed with the first VCO. This would offset the signal in the shape of the ADSR. I thought about adding a module that just returns zeros. But there was some trouble with that getting created on the right device. So, I vote we remove this and don't disable modules -- or if we want to then we can come up with another way to have variable architectures?

Reply:
okay to remove